app/classes/api2/observation_api.rb

```diff
@@ -176,7 +176,7 @@ def add_field_slip_code(observation)
 
         field_slip.update!(observation:)
```

Comment:
Should it also check for `field_slip.user` and do repairs even when a FieldSlip already exists? (lines 175-177)
That would should be belt-and-suspenders after Rails Runner script is created and run.